src/ios/CDVWKWebViewEngine.m

```diff
@@ -123,6 +123,32 @@ - (void)pluginInitialize
                name:UIApplicationWillEnterForegroundNotification object:nil];
 
     NSLog(@"Using WKWebView");
+
+    [self addURLObserver];
+}
+
+- (void)onReset {
+    [self addURLObserver];
+}
+
+static void * KVOContext = &KVOContext;
+
+- (void)addURLObserver {
+    if(![[NSProcessInfo processInfo] isOperatingSystemAtLeastVersion:(NSOperatingSystemVersion){.majorVersion = 9, .minorVersion = 0, .patchVersion = 0 }]){
```

Comment:
To make it simpler, you can use `IsAtLeastiOSVersion(@"9.0")`  (from CDVAvailability.h)
I'm not sure how this whole patch would work for iOS 8, because of this note: https://github.com/apache/cordova-plugin-wkwebview-engine#notes

Is this for a "using a local webserver" context?